src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -193,6 +276,11 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
   const networkBadgebackgroundColor = chainInfo?.backgroundColor
   const filterNetwork = useAtomValue(filterNetworkAtom)
   const tokenDetailData = useTokenDetailQuery(address, chainIdToChainName(filterNetwork))
+  const relevantTokenDetailData = (({ description, homepageUrl, twitterName }) => ({
```

Comment:
this is clever tbh i can't think of a better way to do this without an external library or multiple lines

Reply:
name this `aboutTokenDetailData` since relevant is vague

Reply:
🙏🥲💖